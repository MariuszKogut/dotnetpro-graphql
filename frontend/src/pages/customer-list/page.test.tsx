import React from 'react'
import { render } from '@testing-library/react'
import Page from './page'

jest.mock('react-router-dom', () => ({
  useHistory: () => ({
    history: jest.fn(),
  }),
}))

describe('Page', () => {
  it('renders without errors', () => {
    const { container } = render(<Page />)
    expect(container).toMatchSnapshot()
  })
})
