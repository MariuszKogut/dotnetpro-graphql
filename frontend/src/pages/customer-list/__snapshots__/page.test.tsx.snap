// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Page renders without errors 1`] = `
<div>
  <div
    class="container"
  >
    <div
      class="py-3 row"
    >
      <div
        class="col"
      >
        <h1
          class="pb-3"
        >
          Kundenliste
        </h1>
        <hr />
        <button
          class="btn btn-primary btn-lg"
          type="button"
        >
          Kunde hinzuf√ºgen
        </button>
        <hr />
      </div>
    </div>
    <div
      class="row"
    >
      <div
        class="col"
      >
        <div
          class="spinner-border"
          role="status"
        >
          <span
            class="sr-only"
          >
            Daten werden geladen...
          </span>
        </div>
      </div>
    </div>
  </div>
</div>
`;
